cypress/page/address-step.page.ts

```diff
@@ -2,7 +2,7 @@ class AddressStepPage {
   private proceedCheckoutBtn: string;
 
   constructor() {
-    this.proceedCheckoutBtn = '.cart_navigation span';
+    this.proceedCheckoutBtn = '.cart_navigation .button-medium';
```

Comment:
check if you can use an attribute to indicate we are referring to the processAddress button